<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spons</class>
 <widget class="QDialog" name="spons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid white;
border-radius: 10px;
background: white;

background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgba(72, 145, 217, 255), stop: 1 rgba(255, 255, 255, 255));
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>9</number>
   </property>
   <widget class="QWidget" name="ajouter">
    <attribute name="title">
     <string>ajouter</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>à :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Montant (en Dinar) :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro De Téléphone :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse Email :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_montant">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>251</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mail">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>350</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>145</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>de :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date De Contrat :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom De sponsor :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_debut">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_fin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="afficher">
    <attribute name="title">
     <string>afficher</string>
    </attribute>
    <widget class="QTableView" name="tab_sponsor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>701</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>La Liste des Sponsors :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="modifier">
    <attribute name="title">
     <string>modifier</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom Du Sponsor :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>400</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom_mod">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouvelle Date :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>à :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouveau Montant :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouveau Numéro :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouvelle Adresse Email :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>de :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_montatnt_mod">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_num_mod">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>281</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ad_mail_mod">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_date_debut_mod">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_date_fin_mod">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="supprimer">
    <attribute name="title">
     <string>supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Numéro De Téléphone Du Sponsor</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="num_spon_supp">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>101</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Push_button_supprimer">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>à supprimer :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="chercher">
    <attribute name="title">
     <string>chercher</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Donner Le Nom Du Sponsor : </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sponsor_recherche">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rechercher">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>rechercher</string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_recherche">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>711</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Résultat de Recherche</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>trier</string>
    </attribute>
    <widget class="QTableView" name="table_trie">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>681</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_trier">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>cliquer pour trier</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>mailing</string>
    </attribute>
    <widget class="QLineEdit" name="rcpt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>491</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>destinataire</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>55</y>
       <width>651</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>sujet</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>571</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>attachement ici</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mail_pass">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>votre mot de passe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browseBtn">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>90</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse ...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>651</width>
       <height>291</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>tapez votre message ici ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>651</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>envoyer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>imagerie</string>
    </attribute>
    <widget class="QLabel" name="label_pic">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>461</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>PDF</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_pdf">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>221</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>générer un fichier PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>excel</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_export_excel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>351</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>générer un fichier excel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
